src/main/resources/reference/human_g1k_v37.json

```diff
@@ -0,0 +1,720 @@
+{
+    "contigs": [
+        {
+            "name": "1",
+            "length": 249250621,
+            "isX": false,
```

Comment:
Couldn't you just encode the X and Y chromosomes in lists instead of these booleans everywhere? Same with PAR